tests/test_logger.py

```diff
@@ -0,0 +1,26 @@
+import contextlib
+import io
+import json
+import os
+from unittest.mock import patch
+
+import woodchipper
+import woodchipper.configs
+
+
+def test_bind_log_prefix():
+    with patch.dict(os.environ, dict(WOODCHIPPER_KEY_PREFIX="footest")):
+        buf = io.StringIO()
+        with contextlib.redirect_stdout(buf):
+            woodchipper.configure(config=woodchipper.configs.Minimal, facilities={"": "INFO"})
+            logger = woodchipper.get_logger(__name__)
+            logger.info("Message one.", a=1)
+            logger_with_ctx = logger.bind(**{"b": 2, "customprefix.bar": "baz"})
+            logger_with_ctx.info("Message two.", a=1)
+        messages = [json.loads(message) for message in buf.getvalue().strip().split("\n")]
```

Comment:
This is super cool. I've never seen this technique to capture stdout before. Nice util from contextlib.

@evanmags we were talking about something like this before. Really lightweight approach that I think would get close to what you were thinking about.

Reply:
🎉